test/integration_test.sh

```diff
@@ -142,7 +142,7 @@ fi
 if [ $pipeline_tools_mode == "github" ]; then
   if [ $pipeline_tools_version == "latest_released" ]; then
     printf "\n\nDetermining latest released version of pipeline-tools\n"
-    pipeline_tools_version=$(python $script_dir/get_latest_release.py --repo HumanCellAtlas/pipeline-tools)
+    pipeline_tools_version=$(python $script_dir/get_latest_release.py --repo HumanCellAtlas/pipeline-tools --tag_prefix v)
```

Comment:
I found that only Lira's releases don't have the prefix "v", do we want to add the prefix to Lira's releases, so that we don't need this parameter here?

Reply:
Oh, even we add the "v" to Lira releases, we still need this parameter for pipelines, nevermind.

Reply:
Good point, I edited the Lira releases so they have the v prefix now. Will fix the script once you've finished reviewing.

Reply:
I still think it's a good change to make 'v' the default prefix to use, so we don't have to pass it in for most uses. Will push that change shortly.

Reply:
Thanks! Also adding the "v" to lira's releases result in new docker builds on quay: https://quay.io/repository/humancellatlas/secondary-analysis-lira?tab=tags, so later we next time we deploy Lira we need to change the docker tag carefully in deployment.yml